scripts/events.py

```diff
@@ -875,24 +773,13 @@ def handle_focus(self):
                     )
                 )
 
-            if warrior_amount > 1:
-                focus_text = i18n.t(
-                    "hardcoded.multi_prey_herbs",
-                    warrior_amount=warrior_amount,
-                    count=herb_amount,
-                )
-            elif warrior_amount == 1:
-                focus_text = i18n.t(
-                    "hardcoded.one_prey_herbs",
-                    warrior_amount=warrior_amount,
-                    count=herb_amount,
-                )
-            elif warrior_amount <= 0:
-                focus_text = i18n.t(
-                    "hardcoded.no_prey_herbs",
-                    warrior_amount=warrior_amount,
-                    count=herb_amount,
-                )
+            focus_text = i18n.t(
+                "hardcoded.focus_prey",
+                count=warrior_amount
+            )
+
+            if herb_focus_text:
+                focus_text += f" {herb_focus_text}"
```

Comment:
I really don't like this (it forces other languages to follow the format of two separate sentences in that order), but it's okay.